pkg/plugins/platform/local_test.go

```diff
@@ -292,6 +292,27 @@ func TestLocalStop(t *testing.T) {
 	}
 }
 
+// TestDescribeStopped tests that the describe command works properly on a stopped site.
+func TestDescribeStopped(t *testing.T) {
+	assert := assert.New(t)
+	app, err := GetPluginApp("local")
+	assert.NoError(err)
+
+	for _, site := range TestSites {
+		cleanup := site.Chdir()
+
+		testcommon.ClearDockerEnv()
+		err := app.Init(site.Dir)
+		assert.NoError(err)
+
+		out, err := app.Describe()
+		assert.NoError(err)
+
+		assert.Contains(out, "stopped", "describe correctly reports stopped on a non-running site.")
```

Comment:
If i read [this](https://github.com/stretchr/testify/blob/master/assert/assertions.go#L621) right, seems like `msgAndArgs` is only output on a fail of the assertion, so we would want the message to further explain the failed assertion.